dataproc/src/test/java/CreateClusterTest.java

```diff
@@ -60,35 +60,32 @@ public static void checkRequirements() {
 
   @Before
   public void setUp() {
-    clusterName = String.format("%s-%s", BASE_CLUSTER_NAME, UUID.randomUUID().toString());
-
     bout = new ByteArrayOutputStream();
     System.setOut(new PrintStream(bout));
+    standardOutOrig = System.out;
```

Comment:
You want to set this before you override it on L64 (but you don't need to set it - you never set it back to the original value, so this is unnecessary)

Reply:
I believe I set it back on line 79 with `System.setOut(standardOutOrig);` (or at least, that is my intent with this code)